index.csv

```diff
@@ -1,3 +1,4 @@
+@andrewbranch/untar.js,MIT,1.0.0,1.0.1,1.0.2,1.0.3
```

Comment:
Some weirdness here. The npm package publisher is Andrew Branch, the README says it was taken from bitjs by Jeff Schiller, and the copyright notice in the `LICENSE` file is "2015 Kevin Kwok".

This might be the original source, with copyright notices for Google and Kevin Kwok (GH handle antimatter15):

https://github.com/codedread/bitjs/blob/master/LICENSE

MIT feels like a safe conclusion here, and this data package only codes license terms conclusions, not copyright notices or authorship credits. So I've no objection.